jwst/pipeline/calwebb_image3.py

```diff
@@ -56,10 +56,17 @@ def process(self, input):
 
         input_models = datamodels.open(input)
 
-        # Check if input is multiple exposures, as required by some steps
+        # Check if input is single or multiple exposures
         is_container = isinstance(input_models, datamodels.ModelContainer)
         if is_container and len(input_models.group_names) > 1:
 
+            # Set up Level 2c suffix to be used later
+            suffix_2c_end = 'crf'
+            if isinstance(input_models[0], datamodels.CubeModel):
+                suffix_2c_end = 'crfints'
+            asn_id = input_models.meta.asn_table.asn_id
```

Comment:
Is the "asn_id" here the same as the association candidate ID that's passed along from PPS through the asn pool? They are entries like "o001" or "a3001".

Reply:
Yes, that's the one.